mod-source-record-manager-server/src/main/java/org/folio/verticle/consumers/util/MarcImportEventsHandler.java

```diff
@@ -100,8 +107,32 @@ public void handle(JournalService journalService, DataImportEventPayload eventPa
       JournalRecord journalRecord = JournalUtil.buildJournalRecordByEvent(eventPayload,
         journalParams.journalActionType, journalParams.journalEntityType, journalParams.journalActionStatus);
 
-      populateRecordTitleIfNeeded(journalRecord, eventPayload)
-        .onComplete(ar -> journalService.save(JsonObject.mapFrom(journalRecord), tenantId));
+      if (Objects.equals(journalRecord.getEntityType(), PO_LINE) && journalRecord.getError() != null) {
```

Comment:
if the journal record is a po line but getError is null, I was expecting `processJouirnalRecordForOrder` to process the record. 
If there is no error in the journal record, will `getError` return null?

Reply:
Hello @okolawole-ebsco ! I investigated, and it seems like it would be better to add a check on `journalRecord.getError() != null` to the `processJournalRecordForOrder`-method inside ` if (journalRecord.getOrderId() != null) ` case. I think that it would a**void executing script** for the update if there is NO error (for NOT Failed) PO_LINEs. I added it to my PR. Do you have any objections regarding it?

Reply:
The location of the check inside `processJournalRecordForOrder` makes sense to me. Looks good!

Reply:
@okolawole-ebsco thanks!